<launch>
	<!-- run the position_control.cpp -->

	<node pkg="px4_command" type="target_tracking" name="target_tracking" output="screen">

                <rosparam command="load" file="$(find px4_command)/config/target_tracking.yaml" />
	</node>
</launch>
